sql/driver/driver_test.go

```diff
@@ -335,35 +335,70 @@ CREATE TABLE t.kv (
 		t.Fatal(err)
 	}
 
-	rows, err := db.Query("SELECT * FROM t.kv")
-	if err != nil {
+	statementSuffixWithoutWhere := "FROM t.kv"
+	statementSuffixWithWhere := statementSuffixWithoutWhere + "  WHERE k IN ('a', 'c')"
```

Comment:
Wow, those are some verbose names. I don't think the verbosity is doing anything useful. I would actually just get rid of this parameterization.